velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -377,6 +377,110 @@ TEST_F(DateTimeFunctionsTest, yearTimestampWithTimezone) {
           "year(c0)", std::nullopt, std::nullopt));
 }
 
+TEST_F(DateTimeFunctionsTest, week) {
+  const auto week = [&](std::optional<Timestamp> date) {
+    return evaluateOnce<int64_t>("week(c0)", date);
+  };
+
+  EXPECT_EQ(std::nullopt, week(std::nullopt));
+  EXPECT_EQ(0, week(Timestamp(0, 0)));
+  EXPECT_EQ(52, week(Timestamp(-1, 9000)));
+  EXPECT_EQ(40, week(Timestamp(4000000000, 123000000)));
+  EXPECT_EQ(34, week(Timestamp(998474645, 321000000)));
+
+  setQueryTimeZone("Pacific/Apia");
+  EXPECT_EQ(std::nullopt, week(std::nullopt));
+  EXPECT_EQ(52, week(Timestamp(0, 0)));
+  EXPECT_EQ(52, week(Timestamp(-1, 12300000000)));
+  EXPECT_EQ(40, week(Timestamp(4000000000, 123000000)));
+  EXPECT_EQ(34, week(Timestamp(998474645, 321000000)));
+}
+
+TEST_F(DateTimeFunctionsTest, weekDate) {
+  const auto week = [&](std::optional<Date> date) {
+    return evaluateOnce<int64_t>("week(c0)", date);
+  };
+
+  EXPECT_EQ(std::nullopt, week(std::nullopt));
```

Comment:
Please add something like 

auto parseDate = [](const std::string& dateStr) {
    Date returnDate;
    parseTo(dateStr, returnDate);
    return returnDate;
  };

and use the string representation of the Date for the tests to be more readable. e.g.
EXPECT_EQ(0, week(parseDate("1970-01-01")));

Same for the others if possible.